### DISCUSSION TITLE
Extra Option in 36 "backlight ON in docking station (battery charger)" with minimal voltage sensitive level.

### DISCUSSION THREAD

**Discussion by ErikS-web** [egzumer/uv-k5-firmware-custom#150] [Ideas]
With menu 36. `BackLt` the backlight can be set to (OFF/ON/configured Time). 

I also suggest to **ADD** an (adjustable) **Minimum Voltage Level** of 8.00 volts. 

This way you can realize that the backlight remains ON in the docking station (battery charger, and return to Configured Time when you take the radio with you.

And when the radio is placed back in the charger with a low battery charge, the charging current is first used to charge the battery to a high level. Only then will the backlight turn on again.

**Comment by egzumer** [comment]
It doesn't improve anything, doesn't fix any issue. It's just a cosmetic feature. I understand that it may be somewhat attractive for some, but I'm not interested in such changes.

**Comment by ErikS-web** [comment]
_Altenative:_ 

Within Spectrum Analyser the ``Side Button II` does **_toggle backlight_** 

Can that functionality _toggle backlight_ be added as _Available function_  to [`Custom button functions` ](https://github.com/egzumer/uv-k5-firmware-custom/wiki/Button-functions#custom-button-functions) ?
